.github/workflows/build.yaml

```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
```

Comment:
does this mean prod will rebuild on develop?

Reply:
The `tag_with_sha` means that we'll get a separate tag for each commit, so as long as we don't put prod on `latest`, we'll have to manually update the sha to deploy to prod.

Usually I would do a sha tag for merges to develop for QA, and version tags for tagged commits for releases to prod, but this is a small throw-away application that's not going to live long enough for any of that to matter.